sdk/loadtestservice/azure-developer-loadtesting/samples/sample.py

```diff
@@ -29,112 +29,151 @@
 
 from azure.developer.loadtesting import LoadTestingClient
 from azure.identity import DefaultAzureCredential
-from azure.core.exceptions import HttpResponseError
 import time
 
+# can be installed using pip install azure-mgmt-loadtestservice
```

Comment:
it's good to name the samples more specifically about what they're about. Looking at the name `sample.py` you can't tell what the sample will show. 

There should also be a commented section at the top that shows how to run the sample. [Here's](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/textanalytics/azure-ai-textanalytics/samples/sample_alternative_document_input.py#L7) an example